source/extensions/filters/network/rocketmq_proxy/active_message.h

```diff
@@ -20,6 +20,13 @@ namespace Extensions {
 namespace NetworkFilters {
 namespace RocketmqProxy {
 
+static const std::string BrokerName = "broker_name";
+static const std::string ClusterName = "cluster_name";
+static const std::string WriteQueueNum = "write_queue_num";
+static const std::string ReadQueueNum = "read_queue_num";
+static const std::string BrokerId = "broker_id";
+static const std::string Perm = "perm";
```

Comment:
static const string is not allowed, this will lead test flakes due to the static initialization fiasco. Now LowerCaseString takes string_view as constructor parameter. So these have to be `constexpr absl::string_view`.

Reply:
```
external/com_google_protobuf/src/google/protobuf/map.h:240:14: error: no matching function for call to 'ImplicitConvert'
      return ImplicitConvert(t) < ImplicitConvert(u);
             ^~~~~~~~~~~~~~~
external/com_google_protobuf/src/google/protobuf/map.h:214:27: note: candidate function not viable: no known conversion from 'const absl::string_view' to 'std::string_view' (aka 'basic_string_view<char>') for 1st argument
  static std::string_view ImplicitConvert(std::string_view str) { return str; }
                          ^
external/com_google_protobuf/src/google/protobuf/map.h:220:27: note: candidate function template not viable: no known conversion from 'const absl::string_view' to 'const std::string' (aka 'const basic_string<char, char_traits<char>, allocator<char>>') for 1st argument
  static std::string_view ImplicitConvert(const std::string& str) {
                          ^
```

I change to use `constexpr std::string_view` to fix above error, if must be `constexpr absl::string_view`, I would do covert before related func calls.

Reply:
sorry for force push since rebase.